skiko/src/macosX64Main/kotlin/org/jetbrains/skiko/redrawer/MacOsOpenGLRedrawer.kt

```diff
@@ -0,0 +1,245 @@
+package org.jetbrains.skiko.native.redrawer
+
+import kotlinx.cinterop.CPointer
+import kotlinx.cinterop.useContents
+//import kotlinx.coroutines.Dispatchers
+import org.jetbrains.skiko.native.*
+import platform.CoreFoundation.CFTimeInterval
+import platform.CoreGraphics.CGRectMake
+import platform.CoreVideo.CVTimeStamp
+import platform.OpenGLCommon.CGLContextObj
+import platform.OpenGLCommon.CGLPixelFormatObj
+import platform.OpenGLCommon.CGLSetCurrentContext
+import platform.QuartzCore.CAOpenGLLayer
+import platform.QuartzCore.*
+import platform.darwin.dispatch_async
+import platform.darwin.dispatch_get_main_queue
+import kotlin.system.getTimeNanos
+
```

Comment:
Please clean it up, cyrrent AWT remnants make it look awful.

Reply:
Removed many porting leftovers